velox/dwio/common/SelectiveColumnReader.cpp

```diff
@@ -216,6 +216,9 @@ void SelectiveColumnReader::getIntValues(
           VELOX_FAIL("Unsupported value size: {}", valueSize_);
       }
       break;
+    case TypeKind::TIMESTAMP:
```

Comment:
You don't need this

Reply:
This is needed as the output type is Timestamp

Reply:
You cannot use `getIntValues` directly for `Timestamp`, you need to override `TimestampColumnReader::getValues` like here (seems you are trying to create the same file): https://github.com/facebookincubator/velox/blob/e83fe4c988d91c09dab5fbaa90ce6bd2a77523e0/velox/dwio/parquet/reader/TimestampColumnReader.h#L38